docs/docs/tutorials/blocks/index.ipynb

```diff
@@ -102,7 +102,7 @@
     "\n",
     "| Object Type           | Datapane Block |\n",
     "| --------------------- | -------------- |\n",
-    "| pandas DataFrame      | `dp.Table`     |\n",
+    "| pandas DataFrame      | `dp.DataTable` |\n",
```

Comment:
I think perhaps need to explain the block type depends on the df size

Reply:
Done!